config/schedule.rb

```diff
@@ -28,6 +28,11 @@
   rake " lib_jobs:alma_invoice_status_updates"
 end
 
+# Run on production and staging at 11:30 pm EST or 10:30 pm EDT
+every 1.day, at: '3:30 am', roles: [:prod, :staging] do
```

Comment:
@maxkadel this is totally fine, but you could have utilized web, worker or any of the other roles that were set for both production and staging here.  There are just some task we only want to run on the production server.  That is why prod exists.

Reply:
Fixed.